### PR TITLE
Update Core 3.0.0-beta.1 - Closes #748

### LINKED ISSUE #748: Update Lisk Core docs to version 3.0.0-beta.1
## What should be the new content?

The docs need to be updated to support version 3.0.0-beta.1 of Lisk Core.

- [x] Update config structure
- [x] Update links
- [x] Update dependencies

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #748

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -247,6 +245,6 @@ The first digit, `H`, depends on the number of hard forks, and is incremented wi
 
 NOTE: The initial protocol version 1.0 is defined as the version that was implemented by Lisk Core v1.0.0.
 
-For example, the _protocol version_ is used in P2P communication between Lisk Core nodes, in order to determine if the nodes have compatible versions of the Lisk protocol implemented.
+The _protocol version_ is used e.g. in P2P communication between Lisk Core nodes, in order to determine if the nodes have compatible versions of the Lisk protocol implemented.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
The _protocol version_ is used for example, in P2P communication between Lisk Core nodes, in order to determine if the nodes have compatible versions of the Lisk protocol implemented.
```

File: modules/ROOT/pages/management/configuration.adoc [hunk 2]
```diff
@@ -97,7 +100,7 @@ LISK_CONFIG_FILE=<CONFIG_PATH> pm2 start lisk <2>
 Configurations will be loaded in the following order listed below.
 Each configuration will override the previous one:
 
-. Default configuration values of modules and components of the xref:{url_sdk_framework}[Lisk Framework].
+. Default configuration values of modules and components. See xref:{url_sdk_framework}[Lisk Framework].
```
Comment (chris529) [inline modules/ROOT/pages/management/configuration.adoc:103]:
```suggestion
. Default configuration values of modules and components can be found here in xref:{url_sdk_framework}[Lisk Framework].
```

File: modules/ROOT/pages/setup/commander.adoc [hunk 3]
```diff
@@ -159,4 +151,4 @@ In addition it is also recommended to set up a xref:{url_config_logrotation}[log
 If you are not running Lisk Core locally, you will need to follow the xref:{url_config_api_access}[Control API access] guide to enable access.
 
 If all of the above steps have been successfully completed, the next step is to move on to the configuration section.
-If you wish to enable forging or SSL, please see xref:{url_config}[General configuration].
+If you wish to enable forging or SSL, please see xref:{url_config}[configuration guide].
```
Comment (chris529) [inline modules/ROOT/pages/setup/commander.adoc]:
```suggestion
If you wish to enable forging or SSL, please see the xref:{url_config}[configuration guide].
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor points 👍

File: modules/ROOT/pages/management/configuration.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/management/configuration.adoc:103]: Here I would prefer to leave it as it is :)

**Review by chris529 — APPROVED** [review]
👍

**Pull request merged**